### DISCUSSION TITLE
fdroid still stuck on 0.117

### DISCUSSION THREAD

**Discussion by nathaneltitane** [termux/termux-packages#8536] [General]
does it take that long for fdroid to make new builds available?

**Comment by ghost** [comment]
This may take up to week. More, if something go wrong. As F-Droid removed last builds page (https://f-droid.org/wiki/page/com.termux/lastbuild), we don't have way to track its current state.

**Reply by nathaneltitane** [reply]
understood. thank you

**Reply by agnostic-apollo** [reply]
> we don't have way to track its current state

New design has separate pages for stages.

https://monitor.f-droid.org/builds

`termux-app` has already been built.

https://monitor.f-droid.org/builds/build

Next step is deployment, may happen tomorrow.
